encrypted-pii/backend/src/main/java/org/phenotips/data/encrypted/internal/PIISearchStubSync.java

```diff
@@ -47,6 +48,39 @@ public PIISearchStubSync()
         super("patient-birthyear-updater", new PatientChangingEvent());
     }
 
+    private void readDates(BaseObject obj, String fromDateProperty, String toYearProperty) {
```

Comment:
Put lower level methods below the methods that use them. The code should read like a story, with the more generic, high level methods on top, and the details at the bottom.